### ISSUE TITLE
Add shadow blur to anti-alias. 

### ISSUE THREAD

**Issue by akira-cn** [oframe/ogl#57]
![](https://p4.ssl.qhimg.com/t0162281d413b1a0b1d.jpg)

Might be we should make shadow a bit more smooth.

**Comment by gordonnl** [comment]
Yes! I totally agree. There are quite a few different approaches, each with different pit-falls, and from what I understand they're all quite expensive compared to the current, basic implementation.

From the quick research I've done earlier, I'd like to see an exponential shadow map implementation - it seems to have nice results for relatively few side-effects.

And on the same topic, I'd also love to see an implementation of cascade shadow maps (for larger scenes)!
